packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -35,104 +34,111 @@ Places we can store stored schema information:
-(which is a great place for different applications using the same document to store their data if they want to be less formal about versioning, and do best effort interop with each-other's metadata).
+Note that while many apps might do mostly one or the other, a reasonable design pattern is to take one approach for some data, and the other approach for the rest.
+For example, the core parts of a document might take a schema-on-write approach but allow extensible metadata that is handled with schema-on-read. This is a great place for different applications using the same document to store their data if they want to be less formal about versioning, and do best effort interop with each other's metadata.
 
 # Example/Proposed Design
 
 This design shows an easy to implement MVP, and is not intended to have document or API compatibility with what we do longer term.
-It minimally implements the requirements from "What options to support" while being extensible in the future.
-It can be subsetted for use in earlier milestones, and could be extended for later ones, see "Schedule" below.
+
+It minimally implements the requirements from [Options to Support](#options-to-support) while being extensible in the future.
+It can be subsetted for use in earlier milestones, and could be extended for later ones, see [Schedule](#schedule).
 
 This design is intentionally minimal while still being useful.
 Features not needed for minimal strongly typed application use or the JSON and XML domain schema are not included.
 
 ## Schema Representation
 
-The MVP should include a simple schema system, usable as a MVP for both `stored schema` and `view schema`.
+The MVP should include a simple schema system, usable as an MVP for both `stored schema` and `view schema`.
 
-One possible such schema system is included in [Schema.ts](./Schema.ts).
+One possible such schema system is included in [schema.ts](./schema.ts).
 
 These `TreeSchema` and `FieldSchema` can be added as `stored schema` as part of an edit op,
-which is considered conflicted if it tries to add a type that has a conflicting `name` and is not equal to the existing one.
+which is considered conflicted if it tries to add a type that has a conflicting `name` and is not equal to the existing one. TODO: This was confusing to me. This might be a bit more clear as part of the "Use as `Stored Schema` section.
 
-While a schema must be added to the document as a stored schema to use the type
+While a schema must be added to the document as a stored schema to use the type TODO: Which type? I.e. does this mean the same thing as 'a schema must be added to the document as a stored schema in order to be applied to a specific type of node'?
 (otherwise adding a new type could break existing data which might not even be downloaded on the current client),
 it's possible to add a schema that is compatible with all possible data (assuming the children themselves are compatible with their own types).
+
```

Comment:
I think adding this line break is confusing because in the sentence below it the use of "this" is referring to what's described in the above sentence, and not this section as a whole.

```suggestion
```